db/schema.rb

```diff
@@ -87,9 +90,9 @@
   create_table "notes", force: :cascade do |t|
     t.text "comment"
     t.string "rate"
-    t.bigint "user_id"
-    t.bigint "author_id"
-    t.bigint "company_id"
+    t.integer "user_id"
+    t.integer "author_id"
+    t.integer "company_id"
```

Comment:
why? this should be bigint

Reply:
Strange, I didn't create a migration to change these fields :thinking:
Let me check this